requirements/runtime.txt

```diff
@@ -1,6 +1,6 @@
 future
 matplotlib
-mmcls
+mmcls <= 0.20.1
```

Comment:
Since you request a minimum version of  `mmcls` in `mminstall.txt`, I suggest keeping the consistency here.

```suggestion
mmcls >= 0.19.0, <= 0.20.1
```